main.js

```diff
@@ -0,0 +1,71 @@
+1- const UserChoice = function (num) {
+    if (num === 'rock' || num === 'paper' || num === 'scissors') {
+     return num;
+   } else {
+     console.log('lose');
+   }
+ }
+ 
+ const ComputerChoice = function (num) {
+   
+   
+   switch (Math.floor(Math.random() * 3)) {
+     case 0:
+       return 'rock';
+     case 1:
+       return 'paper';
+     case 2:
+       return 'scissors';
+ }
+ }
+ 
+ function determineWinner(UserChoice, ComputerChoice) {
+ if (UserChoice === ComputerChoice) {
```

Comment:
JS naming convention is camelCase, the first word should start with a small letter. `userChoice`